packages/ember-htmlbars/tests/integration/binding_integration_test.js

```diff
@@ -123,57 +120,6 @@ QUnit.test('should cleanup bound properties on rerender', function() {
   equal(view.$().text(), 'wycats', 'rendered binding');
 });
 
-QUnit.test('should update bound values after view\'s parent is removed and then re-appended', function() {
```

Comment:
Should this really be removed or updated to work without using `Ember.ContainerView`?